bin/metrics-influxdb.rb

```diff
@@ -63,24 +63,23 @@ def parse_output
 
   def check_status
     data = []
-    data.push(create_point(@event['client']['name'], @event['check']['status'], @event['client']['timestamp']))
+    data.push(create_point(@event['check']['name'], @event['check']['status'], @event['client']['timestamp']))
```

Comment:
The change was due to it being a mistake during the first commit. Using the
initial commit there was no indication what check the data point was
actually related to. I didn't had it in the change log as this behavior was
the original intent.
On Mar 25, 2016 8:33 AM, "Mathias Bogaert" <notifications@github.com> wrote:

> In bin/metrics-influxdb.rb
> <https://github.com/sensu-plugins/sensu-plugins-influxdb/pull/18#discussion_r57440309>
> :
>
> > @@ -63,24 +63,23 @@ def parse_output
> >
> >    def check_status
> >      data = []
> > -    data.push(create_point(@event['client']['name'], @event['check']['status'], @event['client']['timestamp']))
> > +    data.push(create_point(@event['check']['name'], @event['check']['status'], @event['client']['timestamp']))
>
> why this change? it's not documented in the changelog...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sensu-plugins/sensu-plugins-influxdb/pull/18/files/d260910c419284587affd0f081fbea358648dadc#r57440309>
>